tsconfig.json

```diff
@@ -15,7 +15,12 @@
     "module": "esnext",
     "moduleResolution": "node",
     "plugins": [
-      { "name": "typescript-plugin-css-modules" }
+      {
+        "name": "typescript-plugin-css-modules"
+      },
+      {
+        "name": "next"
+      }
```

Comment:
I started using the `app` router for the API pages and next decided to introduce some config!